src/components/TreesMap/TreesMap.tsx

```diff
@@ -392,6 +434,37 @@ export const TreesMap = forwardRef<MapRef, TreesMapPropsType>(function TreesMap(
         },
       });
 
+      if (SHOW_3D_TREES) {
```

Comment:
Is this a env var? or why is it uppercase?

Reply:
i made it uppercase to demonstrate that it is taken from an env var: `const SHOW_3D_TREES =
  (process.env.NEXT_PUBLIC_SHOW_3D_TREES ?? 'false') === 'true';`